src/cljs/pyregence/pages/near_term_forecast.cljs

```diff
@@ -530,6 +530,7 @@
   (go
     (!/set-state-legend-list! [])
     (reset! !/last-clicked-info nil)
+    (js/gtag "event" "select-forecast" (clj->js {:forecast-type (str key)}))
```

Comment:
We could use other names in replacement of `select-forecast` and `forecast-type`
Also we could decide on wether use `(str key)` , `(name key)` or any other mapping for the values.
From what I read and tested it seems we really need to pass a map (js obj) here otherwise it will not work.
Example that does not work: `(js/gtag "event" "select-forecast" (str key))`